acropolis-persistence/src/main/kotlin/org/ephyra/acropolis/persistence/api/entity/ApplicationSoftwareEntity.kt

```diff
@@ -0,0 +1,28 @@
+package org.ephyra.acropolis.persistence.api.entity
+
+import javax.persistence.*
+
+/**
+ * Entity to model application software.
+ *
+ * Definition of application software
+ *  Application software is computer software designed to perform a group of coordinated functions, tasks, or activities for the benefit of a user.
+ *
+ * Examples of application software
+ *  - A web browser
+ *  - A word processor
+ *  - An interactive shell or terminal
+ */
+@Entity
+class ApplicationSoftwareEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    var id: Int = 0
+
+    var name: String = ""
```

Comment:
Do we want to introduce any validations? `@NotNull`?

Reply:
Yes absolutely, but I don't know the javax.persistence API so one of the things to do before this PR is ready is go through some documentation and get the persistence tidied up into a nice example for future work

Reply:
Yeah there are annotations that can help describe the data model to the framework. It's the standard javax.persistence API that's implemented by Hibernate and wrapped up in Springy stuff. I saw you picked up 'datastore' so if you get to this before I do then that should be the place to look :)